doc/apidoc/smac.epm.rfr_imputator.rst

```diff
@@ -0,0 +1,7 @@
+smac\.epm\.rfr\_imputator module
+================================
+
+.. automodule:: smac.epm.rfr_imputator
```

Comment:
I've just pushed a commit to this branch adding some more documentation to the BaseImputor class. See 0d9c36e